pkg/logs/input/journald/tailer.go

```diff
@@ -31,7 +32,7 @@ type Tailer struct {
 	source     *config.LogSource
 	outputChan chan *message.Message
 	journal    *sdjournal.Journal
-	blacklist  map[string]bool
+	blacklist  map[string]map[string]bool // a map of string to a "set" of strings
```

Comment:
```suggestion
	blacklist  map[string]map[string]bool // blacklist[key][value] -> true, perhaps?
```
Also, and I realize you didn't introduce this issue, but since the value of the map is never false, this could be `map[string]map[string]struct{}`.  The `struct{}` idiom better suggests "set" instead of "map".